src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerHost.cs

```diff
@@ -53,7 +53,7 @@ internal sealed class DesignerHost : Container, IDesignerLoaderHost2, IDesignerH
         private readonly Dictionary<IComponent, IDesigner> _designers;  // designer -> component mapping
         private readonly EventHandlerList _events; // event list
         private DesignerLoader _loader; // the loader that loads our designers
-        private ICollection _savedSelection; // set of selected components saved across reloads
+        private List<string> _savedSelection; // set of selected components saved across reloads
```

Comment:
This comment seems a little misleading. Maybe change it to the following
```suggestion
        private List<string> _savedSelection; // set of selected component names saved across reloads
```